.github/workflows/check-multi-package.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Check Multi-package
+
+on:
+  pull_request:
+    branches:
+      - karimeldegwy/*
+    paths:
+      - 'SPECS/**'
+      - 'SPECS-EXTENDED/**'
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
why is write permission required for PRs?

Reply:
This is the perm needed to assign someone as a reviewer to the pr https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#request-reviewers-for-a-pull-request